src/services/railgun/wallets/__tests__/balance-update.test.ts

```diff
@@ -59,11 +59,11 @@ describe('balance-update', () => {
       throw new Error('Expected railgunWalletInfo');
     }
     await loadProvider(
-      MOCK_FALLBACK_PROVIDER_JSON_CONFIG_GOERLI,
-      NetworkName.EthereumGoerli,
+      MOCK_FALLBACK_PROVIDER_JSON_CONFIG_MUMBAI,
```

Comment:
Ethereum Goerli seems to be down/dead, so we're using Polygon Mumbai, another testnet.